ui/app/AppLayouts/Chat/popups/community/TransferOwnershipPopup.qml

```diff
@@ -1,87 +1,75 @@
 import QtQuick 2.12
 import QtQuick.Controls 2.3
+import QtQuick.Layouts 1.14
 
 import StatusQ.Core 0.1
 import StatusQ.Core.Theme 0.1
 import StatusQ.Controls 0.1
 import StatusQ.Popups 0.1
 
-
 import utils 1.0
 import shared.controls 1.0
 import shared 1.0
 
 StatusModal {
-    id: popup
+    id: root
 
     property string privateKey
     property var store
 
-    //% "Transfer ownership"
-    header.title: qsTrId("transfer-ownership")
+    header.title: qsTr("Transfer ownership")
+    padding: 16
 
-    onClosed: {
-        popup.destroy();
-    }
+    ColumnLayout {
+        anchors.fill: parent
 
-    contentItem: Item {
-        width: popup.width
-        implicitHeight: Math.max(300, content.height + 32)
-        Column {
-            id: content
-            anchors.top: parent.top
-            anchors.topMargin: 16
-            anchors.horizontalCenter: parent.horizontalCenter
-
-            width: popup.width - 32
-            spacing: 16
-
-            StatusInput {
-                property string elidedPkey: popup.privateKey.substring(0, 15) + "..." + popup.privateKey.substring(popup.privateKey.length - 16)
-
-                id: pKeyInput
-                anchors.left: parent.left
-                anchors.right: parent.right
-                leftPadding: 0
-                rightPadding: 0
-                label: qsTr("Community private key")
-                input.text: elidedPkey
-                input.edit.onActiveFocusChanged: {
-                    pKeyInput.input.text =  pKeyInput.input.edit.focus ? popup.privateKey : elidedPkey
-                }
-                input.rightComponent: StatusButton {
-                    anchors.verticalCenter: parent.verticalCenter
-                    border.width: 1
-                    border.color: Theme.palette.primaryColor1
-                    size: StatusBaseButton.Size.Tiny
-                    text: qsTr("Copy")
-                    onClicked: {
-                        text = qsTr("Copied")
-                        popup.store.copyToClipboard(popup.privateKey)
-                    }
+        spacing: 16
+
+        StatusInput {
+            id: pKeyInput
+
+            Layout.fillWidth: true
+
+            readonly property string elidedPkey: root.privateKey.substring(0, 15) + "..." + root.privateKey.substring(root.privateKey.length - 16)
```

Comment:
In Utils.qml there is a funtion getElidedCompressedPk with very same funtionality. Can you reuse it ?